### DISCUSSION TITLE
When tls is enabled , why is the secret created of type opaque and not  kubernetes.io/tls. will the same secret work with the ingress

### DISCUSSION THREAD

**Discussion by nikhil-rajan-neu** [SeleniumHQ/docker-selenium#2324] [General]
![image](https://github.com/user-attachments/assets/7826839c-02ee-4d7b-9acb-89e6e2708503)

**Comment by VietND96** [comment]
There is a breaking change in the next release, it will be `kubernetes.io/tls` Secret

**Reply by VietND96** [reply]
However, as I tried before, a generic Secret can also work, just need to provide 2 mandatory config keys `tls.crt` and `tls.key`

**Reply by VietND96** [reply]
Chart version `0.33.0` with the update.
